buildSrc/build.gradle.kts

```diff
@@ -29,7 +29,10 @@ tasks.withType<JavaCompile>().configureEach {
 
 dependencies {
     implementation("commons-codec:commons-codec:1.15")
-    implementation("io.github.bonigarcia:webdrivermanager:3.7.1")
+    implementation("io.github.bonigarcia:webdrivermanager:5.1.0") {
+        exclude("com.fasterxml.jackson.core")
+    }
+    implementation("org.seleniumhq.selenium:selenium-java:4.1.2")
```

Comment:
It would be better not bring this to the build classpath.

Reply:
Without selenium the task fails with...
```
Caused by: java.lang.NoClassDefFoundError: org/openqa/selenium/Capabilities
        at org.zaproxy.gradle.tasks.DownloadWebDriver$Download.getInstance(DownloadWebDriver.java:144)
        at org.zaproxy.gradle.tasks.DownloadWebDriver$Download.execute(DownloadWebDriver.java:117)
```

Reply:
Not sure I know what to do to make that happen, anything you can point me too?

Reply:
It could be done with `JavaExec` or the Worker API (might be easier with the latter, with classloader or process isolation).